project/app/api/database.py

```diff
@@ -58,135 +92,158 @@ def close(self):
+        cursor.execute(query)  # Execute Query
+        result = cursor.fetchall()  # fetches all records, we can limit # of records by put LIMIT statement in query
+        conn.close
         return result
 
     def fetch_all_records(self):
+        #conn, cursor = self.conn_curs()
+        conn = self.connection
         cursor = self.connection.cursor()
-        # 'ID','country','province','district','district_id','sector','sector_id','cell','cell_id','village','village_id','name','project_code','type','stage','sub_stage','individuals_directly_served','span','lat','long','form','case_safe_id','opportunity_id','inc_income','inc_income_rwf','inc_income_usd','bridge_image']
-        query = f"""SELECT "ID", country, province, district,district_id,sector,sector_id,cell,cell_id,village,village_id,name,project_code,type,stage,sub_stage,individuals_directly_served,span,lat,long,form,case_safe_id,opportunity_id,inc_income,inc_income_rwf,inc_income_usd,bridge_image FROM public."Bridges"  """
-        cursor.execute(query)
-        result = cursor.fetchall()
-        columns = ['ID', 'country', 'province', 'district', 'district_id', 'sector', 'sector_id', 'cell', 'cell_id', 'village', 'village_id', 'name', 'project_code', 'type', 'stage',
-                   'sub_stage', 'individuals_directly_served', 'span', 'lat', 'long', 'form', 'case_safe_id', 'opportunity_id', 'inc_income', 'inc_income_rwf', 'inc_income_usd', 'bridge_image']
-        # columns = ['ID', 'country']
-        df = pd.DataFrame(result, columns=columns)
-        df_json = df.to_json(orient='records')
-        parsed = json.loads(df_json)
-        return parsed
-
-    def fetch_query_given_project(self, project_code: int):
-        cursor = self.connection.cursor()
-        query = f"""SELECT * FROM public."Bridges" where project_code={project_code} """
+        query = """ SELECT * from public."B2P_oct_2018";"""
         cursor.execute(query)
         result = cursor.fetchall()
-        columns = ['ID', 'country', 'province', 'district', 'district_id', 'sector', 'sector_id', 'cell', 'cell_id', 'village', 'village_id', 'name', 'project_code', 'type', 'stage', 'sub_stage', 'individuals_directly_served', 'span', 'lat', 'long', 'community_served_1', 'community_served_1_id',
-                   'community_served_2', 'community_served_2_id', 'community_served_3', 'community_served_3_id', 'community_served_4', 'community_served_4_id', 'community_served_5', 'community_served_5_id', 'form', 'case_safe_id', 'opportunity_id', 'inc_income', 'inc_income_rwf', 'inc_income_usd', 'bridge_image']
-        df = pd.DataFrame(result, columns=columns)
+        #conn.close()
+        df = pd.DataFrame(result, columns=['index'] + self.columns)
+        df = df.iloc[:, 1:]  # skip 1 column, as it corresponds to an index column
         df_json = df.to_json(orient='records')
         parsed = json.loads(df_json)
         return parsed
 
```

Comment:
This query is referencing a capitalized column header, As I mentioned above the dataset we are using has all lowercase column headers. - If this is an issue when querying it looks like this needs to be organized.

Reply:
This is the old column format it will be updated, if I remember what I said, I am working on. the uncleaned data set, and this follows my column format.